### PR TITLE
Lead condition question

### PR DESCRIPTION
**Pull request by atusa-mca**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 API_LEAD_APP_REQUEST=https://7a96bd7aebc4.ngrok.io/lead-srv
 API_SESSION_APP_REQUEST=https://7a96bd7aebc4.ngrok.io/session-srv
-API_USER_APP_REQUEST=https://7a96bd7aebc4.ngrok.io/user-srv
+API_USER_APP_REQUEST=hhttps://7a96bd7aebc4.ngrok.io/user-srv
 API_REGION_APP_REQUEST=https://7a96bd7aebc4.ngrok.io/region-srv
```
Comment (ali-mca) [inline .env]: change it to https

File: lib/models/entities/lead/lead.dart [hunk 2]
```diff
@@ -102,34 +102,29 @@ class Lead {
...
+      required this.city,
+      required this.state,
+      required this.zipCode,
+      required this.region});
+
+  factory Lead.fromJson(Map<String, dynamic> json) => _$LeadFromJson(json);
 
   Map<String, dynamic> toJson() => _$LeadToJson(this);
-
 }
```
Comment (ali-mca) [inline lib/models/entities/lead/lead.dart]: Revert back the changes in this file.

File: lib/views/lead/lead_schedule_view.dart [hunk 3]
```diff
@@ -295,24 +290,24 @@ class _LeadScheduleViewState extends State<LeadScheduleView> {
...
+                      child: FxTimeText(
+                          label: 'Schedule Time',
                           initValue: TimeOfDay.now(),
                           onTimeChanged: (newTime) => scheduleTime = newTime,
-                          validator: (value) => _validate(value))
-                  ),
+                          validator: (value) => _validate(value))),
                 ],
               ),
             ),
```
Comment (ali-mca) [inline lib/views/lead/lead_schedule_view.dart]: Would you please revert back changes in this file and the next one.

**Review by ali-mca — CHANGES_REQUESTED** [review]

File: lib/views/lead/lead_schedule_view.dart (hunk 3 above)
Comment (atusa-mca) [reply lib/views/lead/lead_schedule_view.dart]: It's fixed

File: lib/models/entities/lead/lead.dart (hunk 2 above)
Comment (atusa-mca) [reply lib/models/entities/lead/lead.dart]: It's updated

File: ios/Podfile.lock [hunk 4]
```diff
@@ -31,4 +31,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: aafe91acc616949ddb318b77800a7f51bffa2a4c
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.10.1
```
Comment (ali-mca) [inline ios/Podfile.lock]: Don't commit Podfile.lock

**Review by ali-mca — CHANGES_REQUESTED** [review]
Exclude .DS_Store from pull request,

File: .env (hunk 1 above)
Comment (atusa-mca) [reply .env]: It's fixed

**Pull request merged**